src/components/FormCK/FormCK.js

```diff
@@ -0,0 +1,196 @@
+                            id="email"
+                            type="email" 
+                            name="email"
+                            ref={register({ 
+                                required: "⚠️ E-mail is required",
+                                pattern: {
+                                    value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i,
+                                    message: "⚠️ Invalid email address"
+                                }
+                            })}
+                            placeholder="Your email address"
+                        />
+                        {errors.email && <span>{errors.email.message}</span>}
+
+                        <input 
+                            className="stashaway" 
+                            autoComplete="nope" 
+                            type="text" 
+                            id="address" 
+                            name="address" 
+                            ref={register}
+                            placeholder="Your address here" 
+                        />
+                        <button type="submit" disabled={isSubmitting}>{submitText}</button>
+                        {/* <button  disabled={state.submitting}>
+                            Sign Up
+                        </button> */}
+                        <p>No spam. Unsubscribe at any time. <span role="img" aria-label="ok">✌️</span></p>
+                    </Box>
+                    </>
+                )}
+            </div>
+        </FormCkWrapper>
+    )
+}
+
+FormCK.defaultProps = {
+    children: <DefaultLeftCopy />,
+    copyBefore: <DefaultBeforeCopy />,
+    submitText: "Improve my career ?",
+};
+
+const getDefaultFormId = graphql`
+    query {
+        site {
+            siteMetadata {
+                convertkit {
+                    defaultFormId
+                }
+            }
+        }
+    }
+`;
+
+const FormCkWrapper = styled.div`
+
+    .copy-content {
+        display: grid;
+        grid-template-columns: repeat(auto-fit,minmax(200px,1fr));
+
+        box-shadow: 0 2px 15px 0 rgba(210,214,220,0.5);
+        overflow: hidden;
+
+        .copy-left {
+            background-color: #f9fafb;
+            padding: 2rem;
+            text-align: center;
+        }
+
+        form {
+            display: flex;
+            flex-direction: column;
+
+            padding: 2rem;
+            text-align: center;
+
+                input {
+                    border-color: #e3e3e3;
+                    border-radius: 4px;
+                    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+                    font-size: 1rem;
+                    font-weight: 400;
+                    line-height: 1.4;
+                    margin-top: 1rem;
+                    padding: 0.8rem;
+                }
+
+                span {
+                    color: red;
+                    text-align: left;
+                }
+
+                .stashaway {
```

Comment:
Let's make this `.required` so bots get even more confused

Reply:
Nice one